src/plugins/ncr/screens/node/index.jsx

```diff
@@ -208,9 +210,28 @@ export default class AbstractNodeScreen extends React.Component {
   }
 
   renderBody() {
-    const { delegate, getNodeRequestState } = this.props;
+    const { delegate, getNodeRequestState, nodeRef } = this.props;
 
     if (!delegate.getNode()) {
+      if (!this.nodeAlertTimeoutId) {
+        this.nodeAlertTimeoutId = setTimeout(() => {
+          const nodeLabel = nodeRef.getLabel().charAt(0).toUpperCase() + nodeRef.getLabel().slice(1);
```

Comment:
@aloncohenn this is gonna break down when you have something that is hyphenated, not worth it, just use the raw label